### DISCUSSION TITLE
Reader Error Unable to Determine Path

### DISCUSSION THREAD

**Discussion by colefreeman** [great-expectations/great_expectations#6056] [Support] [answered]
I am following the getting started tutorial, but continually get this reader error. I am not sure exactly what is going on here...my path is correct to this file. Can anyone please help, I have been stuck here for a while.

Unable to determine reader method from path: "C:\DataEngineeringPrograms\fakerData\great_expectations\..\people.CSV".

**Comment by kyleaton** [comment, marked as answer]
Can you try setting .CSV to lowercase -> .csv 

If this still doesn't work could you include your datasource config and batch request? 

cc @talagluck

**Reply by colefreeman** [reply]
changing the file name from people.CSV to people.csv worked...thank you!

**Reply by kyleaton** [reply]
Awesome! Glad that helped!
